### DISCUSSION TITLE
Put timestamp before or after っ?

### DISCUSSION THREAD

**Discussion by PlaylistsTrance** [Juna-Idler/RhythmiKaRuTTE#5] [Q&A] [answered]
Not sure where to ask this, hopefully this is a good place. In the line 恋にし`たって`言えそうだけど, where should the karaoke timestamps be placed? Like `[01:24.26]た[01:24.73]って[01:25.10]` or `[01:24.26]たっ[01:24.73]て[01:25.10]`?

**Comment by Juna-Idler** [comment, marked as answer]
`たって` → `TATTE`
In romaji notation, the consonants of the later letters are overlapped, but
Basically, small letters are tied to the letters in front.

`[01:24.26]たっ[01:24.73]て[01:25.10]`

If you want to finely control the scroll speed, put a stamp in between.

`[01:24.26]た[01:24.50]っ[01:24.73]て[01:25.10]`



ローマ字表記だと後の文字の子音を重ねますが、
基本的には、小さい文字は手前の文字に結び付きます。

細かくスクロールの速度を制御したいときは間にもスタンプを入れます。
